lib/types/proposals.ts

```diff
@@ -26,6 +33,8 @@ export type ProposalAccount = MergeWithOptionalFields<
 >;
 
 export type ProposalWithVaults = Proposal & {
-  baseVaultAccount: VaultAccount;
```

Comment:
I don't know if you're associating proposals with their markets somewheres, but the data structure there is certainly going to change with the AMM as there aren't twapmarket and market so it's just all pricingModelPassAcct (what I'm calling it in the DB) and then passMarketAcct

In the case with the AMM only the pricingModelPassAcct will be filled.